### DISCUSSION TITLE
Docs and embedded Images

### DISCUSSION THREAD

**Discussion by Resousse** [facebook/docusaurus#8334] [Q&A] [answered]
Hello,
I have docs hierarchy, which some embeds image in md page, however, I would like to translate my page (this is pretty well explained) and keep the same version of the image asset instead of duplicating the content in the `i18n/fr/docusaurus-plugin-content-docs/current` directory.

`![Modifier le type d'un champs](./changeField.png)`

And I get `Error: Image i18n/fr/docusaurus-plugin-content-docs/current/tuto/02-changefield/changeField.png used in i18n/fr/docusaurus-plugin-content-docs/current/tuto/02-changefield/index.md not found`

Is it possible to reuse the same image asset, that the one used in the default lang version? How?
Is the way to put common assets in `/static/img` the better one?

Thanks

**Comment by slorber** [comment, marked as answer]
Hey

I am aware of this problem, in particular for people translating content on Git (I mean, not using Crowdin or another SaaS), as I do on my own newsletter site https://thisweekinreact.com/


I'd like to add in the future another i18n convention so that you can just write:
- `docs/tuto/02-changefield/index.md`
- `docs/tuto/02-changefield/index.fr.md`

The asset could exist in a single place at `docs/tuto/02-changefield/changeField.png`, and eventually be localized with `changeField.fr.png`

Does it make sense for your use case?

**Reply by duskmoon314** [reply]
May I ask if there is any progress on this? It seems v3.8 still needs to put md under the i18n directory.

**Reply by slorber** [reply]
I've made a POC PR but it's more complex than I thought to implement 😓 

https://github.com/facebook/docusaurus/pull/9700

**Comment by Resousse** [comment]
Yes! it would be awesome! 🥇 
And moreover, will simplify the process of translation of docs!
Thanks!
